Formula/fraggenescan.rb

```diff
@@ -0,0 +1,20 @@
+class Fraggenescan < Formula
+  # cite Rho_2010: "https://doi.org/10.1093/nar/gkq747"
+  desc "Predicting genes in short and error-prone reads"
+  homepage "https://github.com/COL-IU/FragGeneScan"
+  url "https://downloads.sourceforge.net/project/fraggenescan/FragGeneScan1.31.tar.gz"
+  sha256 "cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c"
+
+  depends_on "gcc"
```

Comment:
This dependency on `gcc` shouldn't be necessary. is it addressing a particular error that you're seeing?